plugins/admincmd.sma

```diff
@@ -17,114 +17,79 @@
-new g_Tracker;
-new g_Size;
-
-public Trie:g_tempBans
+new g_szNames[OLD_CONNECTION_QUEUE][MAX_NAME_LENGTH];
+new g_szSteamIDs[OLD_CONNECTION_QUEUE][32];
+new g_szIPs[OLD_CONNECTION_QUEUE][32];
+new g_iAccess[OLD_CONNECTION_QUEUE];
+new g_iTracker;
+new g_iSize;
+
+public Trie:g_tTempBans;
 new Trie:g_tXvarsFlags;
 
-stock InsertInfo(id)
+public plugin_init()
 {
-	
-	// Scan to see if this entry is the last entry in the list
-	// If it is, then update the name and access
-	// If it is not, then insert it again.
-
-	if (g_Size > 0)
+	register_plugin("Admin Commands", AMXX_VERSION_STR, "AMXX Dev Team");
+
+	register_dictionary("admincmd.txt");
+	register_dictionary("adminhelp.txt");
+	register_dictionary("common.txt");
+
+	register_concmd("amx_kick",			"cmdKick",		ADMIN_KICK,					"AMX_KICK_SYNTAX",		.info_ml = true);
+	register_concmd("amx_ban",			"cmdBan",		ADMIN_BAN|ADMIN_BAN_TEMP,	"AMX_BAN_SYNTAX",		.info_ml = true);
+	register_concmd("amx_banip",		"cmdBanIP",		ADMIN_BAN|ADMIN_BAN_TEMP,	"AMX_BANIP_SYNTAX",		.info_ml = true);
+	register_concmd("amx_addban",		"cmdAddBan",	ADMIN_BAN,					"AMX_ADDBAN_SYNTAX",	.info_ml = true);
+	register_concmd("amx_unban",		"cmdUnban",		ADMIN_BAN|ADMIN_BAN_TEMP,	"AMX_UNBAN_SYNTAX",		.info_ml = true);
+	register_concmd("amx_slay",			"cmdSlay",		ADMIN_SLAY,					"AMX_SLAY_SYNTAX",		.info_ml = true);
+	register_concmd("amx_slap",			"cmdSlap",		ADMIN_SLAY,					"AMX_SLAP_SYNTAX",		.info_ml = true);
+	register_concmd("amx_leave",		"cmdLeave",		ADMIN_KICK,					"AMX_LEAVE_SYNTAX",		.info_ml = true);
+	register_concmd("amx_pause",		"cmdPause",		ADMIN_CVAR,					"AMX_PAUSE_SYNTAX",		.info_ml = true);
+	register_concmd("amx_who",			"cmdWho",		ADMIN_ADMIN,				"AMX_WHO_SYNTAX",		.info_ml = true);
+	register_concmd("amx_cvar",			"cmdCvar",		ADMIN_CVAR,					"AMX_CVAR_SYNTAX",		.info_ml = true);
+	register_concmd("amx_xvar_float",	"cmdXvar",		ADMIN_CVAR,					"AMX_XVAR_SYNTAX",		.info_ml = true);
+	register_concmd("amx_xvar_int",		"cmdXvar",		ADMIN_CVAR,					"AMX_XVAR_SYNTAX",		.info_ml = true);
+	register_concmd("amx_plugins",		"cmdPlugins",	ADMIN_ADMIN,				"AMX_PLUGINS_SYNTAX",	.info_ml = true);
+	register_concmd("amx_modules",		"cmdModules",	ADMIN_ADMIN,				"AMX_MODULES_SYNTAX",	.info_ml = true);
+	register_concmd("amx_map",			"cmdMap",		ADMIN_MAP,					"AMX_MAP_SYNTAX",		.info_ml = true);
+	register_concmd("amx_extendmap",	"cmdExtendMap",	ADMIN_MAP,					"AMX_EXTENDMAP_SYNTAX",	.info_ml = true);
+	register_concmd("amx_cfg",			"cmdCfg",		ADMIN_CFG,					"AMX_CFG_SYNTAX",		.info_ml = true);
+	register_concmd("amx_nick",			"cmdNick",		ADMIN_SLAY,					"AMX_NICK_SYNTAX",		.info_ml = true);
+	register_concmd("amx_last",			"cmdLast",		ADMIN_BAN,					"AMX_LAST_SYNTAX",		.info_ml = true);
+	register_clcmd("amx_rcon",			"cmdRcon",		ADMIN_RCON,					"AMX_RCON_SYNTAX",		.info_ml = true);
+	register_clcmd("amx_showrcon",		"cmdShowRcon",	ADMIN_RCON,					"AMX_RCON_SYNTAX",		.info_ml = true);
+	register_clcmd("pauseAc",			"cmdLBack");
```

Comment:
Please use spaces for alignment.

Reply:
Done.

Reply:
@OciXCrom looks perfect now, thanks!
![image](https://user-images.githubusercontent.com/5282741/45426713-ab06aa80-b6a5-11e8-9624-e73e24510334.png)